app/konnect/getting-started/import.md

```diff
@@ -39,6 +40,9 @@ When you provide any {{site.konnect_short_name}} flags, decK targets the `cloud.
     ```sh
     deck gateway ping --konnect-token-file /PATH/TO/FILE
     ```
+    
+    {:.note}
+    > **Note**: If you don't want to pass the PAT in a file, you can either pass the PAT directly in the CLI (`--konnect-token PAT_02uI9CEOkYo36NlJnFVyZf8xDxfgirtgq0NvNWASfweoGMqA`) or via an environmental variable (`export DECK_KONNECT_TOKEN=PAT_02uI9CEOkYo36NlJnFVyZf8xDxfgirtgq0NvNWASfweoGMqA && deck gateway ping`).
```

Comment:
```suggestion
    > **Note**: Alternatively, pass the PAT directly using `--konnect-token PAT_your_token` or set it as an environment variable with `export DECK_KONNECT_TOKEN=PAT_your_token `
```
The current implementation breaks the note block, my suggestions should fix this?
![image](https://github.com/Kong/docs.konghq.com/assets/23319190/5a46c973-e2e9-4532-964c-9b1c0ca066a7)


Outside of this suggestion, offering all three options breaks away from our goal of being opinionated about the way someone should do something. I think we can solve that by changing this into two steps: 

1. Set your PAT as an environment variable 
2. Authenticate w/ Deck Ping. 

then if you still want to showcase the other methods you can create a note like this: 
"Alternatively, you can pass your PAT directly into the CLI using `--konnect-token <pat>, or read it to a file using `--konnect-token-file /PATH/TO/FILE`

Reply:
Or one step
Set your PAT as an environment variable and authenticate 
`export DECK_KONNECT_TOKEN=PAT_02uI9CEOkYo36NlJnFVyZf8xDxfgirtgq0NvNWASfweoGMqA && deck gateway ping`